pandasaurus/query.py

```diff
@@ -225,6 +227,58 @@ def synonym_lookup(self) -> pd.DataFrame:
 
         return result_df
 
+    def get_most_specific_objects(self, predicate: str, ontology: str):
+        """
+
+        Args:
+            predicate: Relationship that wanted to be explored
+            ontology: PURL of obo ontologies in Ubergraph.
+
+        Examples:
+            Example Ontology PURLs:
+                - http://purl.obolibrary.org/obo/cl.owl \n
+                - http://purl.obolibrary.org/obo/uberon.owl
+
+        Returns:
+
+        """
+        subject_list = [term.get_iri() for term in self.__term_list]
+        query_string = get_most_specific_objects_query(subject_list, predicate, ontology)
+        return (
+            pd.DataFrame(
+                [res for res in run_sparql_query(query_string)],
+                columns=["s", "s_label", "p", "o", "o_label"],
+            )
+            .sort_values("s")
+            .reset_index(drop=True)
+        )
```

Comment:
Do we consider this as `enriched_df` as well? I wasn't sure if I should have assigned it to `enriched_df` variable. Same for other method as well CC @dosumis